docs/en/api/POSIX-API.md

```diff
@@ -314,7 +314,7 @@ If the value is too small, FUSE may frequently create and destroy threads which
 Note that, libfuse introduce this mount option in 3.2 while Alluxio FUSE supports libfuse 2.9.X which does not have this mount option.
 
 The Alluxio docker image [alluxio/{{site.ALLUXIO_DOCKER_IMAGE}}](https://hub.docker.com/r/alluxio/{{site.ALLUXIO_DOCKER_IMAGE}}/)
-enables this property by modifying the [libfuse source code](https://github.com/cheyang/libfuse/tree/fuse_2_9_5_customize_multi_threads_v2).
+enables this property by modifying the [libfuse source code](https://github.com/Alluxio/libfuse/tree/fuse_2_9_5_customize_multi_threads).
```

Comment:
Not mention how to install the changed fuse lib?

Reply:
We directly build libfuse 2.9.5 customize version in Alluxio docker image and doesn't recommend user do this